src/jobs/jobs.controller.ts

```diff
@@ -547,76 +629,94 @@ export class JobsController {
 
 
   /**
-   * Find job by id
+   * Get job by id
    */
   @UseGuards(PoliciesGuard)
   @CheckPolicies((ability: AppAbility) =>
-    ability.can(AuthOp.Read, JobClass),
+    ability.can(AuthOp.JobsRead, JobClass),
   )
   @Get(":id")
   @ApiOperation({
-    summary: "It finds an existing job based on its id.",
-    description: "It finds an existing job based on its id.",
+    summary: "It returns the requested job.",
+    description: "It returns the requested job.",
   })
   @ApiResponse({
     status: HttpStatus.OK,
     type: JobClass,
     description: "Found job",
   })
   async findOne(
-    @Req() request: Request,
```

Comment:
You should keep the request.
We use this to retrieve the user and decide if he/she has access to the job